cmd/goquery_completion/conditional.go

```diff
@@ -169,3 +285,127 @@ func conditional(args []string) []string {
 
 	return complete(tokenize, join, next, unknown, last(args))
 }
+
+// verifySuggestion enforces some structural constraints on the condition,
+// only if the current suggestion is the dir keyword, or the dir keyword
+// has already appeared, ensuring that the dir keyword only occurs at valid places.
+func verifySuggestion(sugg suggestion, tokens []string, prev string, openParens, dirKeywordCount int) bool {
+	if strings.Contains(sugg.token, types.FilterKeywordDirection) ||
+		strings.Contains(sugg.token, types.FilterKeywordDirectionSugared) ||
```

Comment:
Nitpick: Is this alignment correct? AFAIK this usually gets aligned with the first condition, not with the inner block... :thinking:

Reply:
I also always thought this alignment looked weird but I think it is correct (`gofmt` does not change it).

Reply:
Alright then, who wants to argue with `gofmt`... :stuck_out_tongue: - Thanks for cross-checking!